pkgs/development/coq-modules/serapi/default.nix

```diff
@@ -71,7 +71,8 @@ in
           if version == "8.11.0+0.11.1" then version
           else builtins.replaceStrings [ "+" ] [ "." ] version
         }.tbz";
-    sha256 = release."${version}".sha256;
+    # abort/syntax error will fail package set eval, but throw is "fine"
```

Comment:
Why this change?

Reply:
because without it, ofborg-eval fails due to niche consequences of lazy evaluation.

basically, evaluating `meta` now evaluates `src`, which causes problems for this package specifically.

Reply:
Rewording it, your `meta.eyecandy` field broke Nixpkgs.

Reply:
@AndersonTorres If you want your criticism to be meaningful I'd suggest not leaving it up for interpretation.

Reply:
How exactly is this open to interpretation? Asking sincerely. 

Creating a `meta.XYZ` field introduced a breaking change.
A breaking change in an otherwise completely unrelated piece of code.

Reply:
I don't consider useful metadata to be eye candy, that's why I'm asking for clarification.

How that relates to the field's relationship to the derivations `src` attribute is the actual question we should be following up.